corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -888,15 +898,47 @@ hqDefine('geospatial/js/models', [
                 );
             }
 
+            loadUserData();
             self.filteredCaseData(self.caseData);
             self.totalItems(self.filteredCaseData().length);
         };
 
+        function loadUserData() {
+            self.userData([]);
+            for (const item of self.mapModel.userMapItems()) {
+                self.userData.push(item.getJson());
+            }
+
+            $('#user-assignment-select').select2({
+                placeholder: gettext('No user selected (unassign mode)'),
```

Comment:
:thinking: Is there a more concise way to say this?
```suggestion
                placeholder: gettext("Don't assign a user"),
```
Or
```suggestion
                placeholder: gettext('No user assigned'),
```
:shrug: Maybe I'm bikeshedding. What you have is fine.

Reply:
I originally had "No user assigned" as the text, however since this mode also allows the user to unassign cases I thought it might make sense to make the text more explicit. 

@kaapstorm How does "Unassign user" sounds? But I'm also happy to go with "No user assigned" if you think that makes more sense.

Reply:
After seeing things in action (thank you for the gifs), I  feel like the selection of mobile worker here seems more like a filter for the underlying table.

What to you think of changes something in line of below to make things more intuitive in terms of UX ?
- Add explicit labels for both selects ,  `Assignment Status`  and `Select worker to assign to`  respectively. Makes UI a bit bloated but worth it ?
- Consider disabling the mobile worker selection unless any case is selected 

I like the `Unassign User` option here considering it is  more of an action.

Reply:
@ajeety4 I quite like the idea of adding labels to the dropdowns at the top. They certainly help with making things a bit more explicit and after some local testing they don't make the UI that much more bloated. I have addressed both points in [ac5de8a](https://github.com/dimagi/commcare-hq/pull/34972/commits/ac5de8a541e88cdb3721508ca2dd5e414509a1ca) and [c1628da](https://github.com/dimagi/commcare-hq/pull/34972/commits/c1628da676da8e72c297b3be16b428da7bcc12c9).

Reply:
Great, thanks for the changes